content/concepts/projects-using-ocean.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: Projects using Ocean Protocol
-description: These are live projects that leverage core functionality of Ocean, such as reading or writing on Ocean datatokens or data pools. They don't rely on Ocean Market for an interface, by either having their own GUI or CLI, or using another app.
+description: These are live projects that use leverage core functionality of Ocean, use Ocean Market, or use OCEAN token with staking and the like.
```

Comment:
> These are live projects that use leverage...

The "leverage" here seems as it was supposed to be deleted?

Reply:
Good eye, fixed